server/startup/migrations/v046.js

```diff
@@ -0,0 +1,21 @@
+RocketChat.Migrations.add({
+	version: 46,
+	up: function() {
+		// Set default message viewMode to 'normal' or 'cozy' depending on the users' current settings and remove the field 'compactView'
+		var normalViewUserIds = [],
+			cozyViewUserIds = [];
+		RocketChat.models.Users.find().forEach(
+			function(user){
+				if(user.settings && user.settings.preferences && user.settings.preferences.compactView){
+					cozyViewUserIds.push(user._id);
+				}else{
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keyword "else" must be followed by whitespace.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2454581114/issues/source?bid=3199634&fileBranchId=3237220#l11)